### DISCUSSION TITLE
Issues with FedEx Tracking

### DISCUSSION THREAD

**Discussion by testinggithubuser490** [bromite/bromite#2203] [Q&A]
Read the readme and the FAQ again and I also searched the discussion albeit quickly, so I'm sorry if this has been answered before. Does anyone have problems with using the FedEx mobile site? I enter my tracking number and just get "Unfortunately we are unable to retrieve your tracking results at this time, please try again later." I have also tried just copying the URL itself rather than manually submitting the tracking number on the website, same story. It's been this way for a few months now, it works ok everywhere else. I've tested with lots of different tracking numbers, and I just loaded up a fresh install of Bromite on a spare Android phone that was factory reset. Running into the same exact thing so I'm thinking it might be more specific to the application than it is my environment - though I could be entirely wrong about that.

**Comment by hdjdb** [comment]
I am also having this issue. Isn't a problem in either Vanadium or Mull (site-specific enhanced tracking protection disabled). My guess is that one of the adblock filters is interfering with their login, but I can't check as I've uninstalled the app since clearly it isn't daily driver material.

**Comment by testinggithubuser490** [comment]
Thanks, at least I'm not alone. I have tried turning off all adblocking and making sure to accept all cookies from the Fedex site [including third party]. Neither worked there so I'm really not sure what it is.

**Reply by Bugaddr** [reply]
Check cookies setting ?

**Reply by Bugaddr** [reply]
Fedex website has compatblity problem i guess, most of the time it don't work

**Reply by Bugaddr** [reply]
If possible test on different browser

**Comment by csagan5** [comment]
It could be due to fingerprinting mitigations; you can use https://developer.chrome.com/docs/devtools/remote-debugging/ to verify what JavaScript errors are there, if any. But in general we will not do troubleshooting in discussions, you have to submit a bug report if you think there is a bug.
